src/backend/logger/logger.ts

```diff
@@ -32,17 +23,42 @@ export enum LogPrefix {
   WineTricks = 'Winetricks'
 }
 
-let longestPrefix = 0
+type LogInputType = unknown[] | unknown
+
+// helper to convert LogInputType to string
+function convertInputToString(param: LogInputType): string {
+  const getString = (value: LogInputType): string => {
+    switch (typeof value) {
+      case 'string':
+        return value
+      case 'object':
+        // Object.prototype.toString.call(value).includes('Error') will catch all
+        // Error types (Error, EvalError, SyntaxError, ...)
+        if (Object.prototype.toString.call(value).includes('Error')) {
```

Comment:
Why the `toString.call`? Wouldn't just
```suggestion
        if (Object.prototype(value).includes('Error')) {
```
work?

Reply:
![Bildschirmfoto vom 2022-09-11 17-34-38](https://user-images.githubusercontent.com/61798668/189536098-7af77074-d254-4e7c-8b0a-1f4c9b7c84bd.png)

I was searching the internet and they suggest the `toString.call`

Reply:
Ah sorry, meant `getPrototypeOf`
```
$ node
Welcome to Node.js v16.16.0.
Type ".help" for more information.
> Object.getPrototypeOf(EvalError()).name
'EvalError'
> Object.getPrototypeOf(EvalError()).name.includes('Error')
true
```

Reply:
Yes it might work, but my way works better on undefiend scenarios. Your let the test fail:
![image](https://user-images.githubusercontent.com/61798668/189543182-96c49d78-bd30-44cc-94f3-e2dca352334d.png)